src/styles/card.css

```diff
@@ -0,0 +1,62 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-name {
+  padding: 40px 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.card__product-code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  line-height: 18px;
+}
+
+.card__price-tag {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  margin-top: 16px;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+
+  background-color: #00acdc;
+  font-size: 14px;
+  font-weight: 700;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.card__button:hover {
+  border-color: #00acdc;
+  background-color: #fff;
+  color: #00acdc;
```

Comment:
you typed the #00acdc 3 times
but what if we decide to use another color?
we would need to find all the places we use it and change
if you use CSS variable you can change the color in 1 place

Reply:
Do I make for example variables.css, and put all the colors I used in there?

Reply:
Do I make for example variables.css, and put all the colors I used in there?
:root {
  --primary-color: #616070;
  --secondary-color: #060b35;
  --blue-color: #00acdc;
  --white-color: #fff;
}
something like this for example